tests/test_adjoint.py

```diff
@@ -90,7 +96,7 @@ def test_adjoint_F(self, mkey, shape, kernel, space_order, time_order, setup_fun
         location from data. This test uses the conventional dot test:
         < Fx, y> = <x, F^T y>
         """
-        tn = 500.  # Final time
+        tn = 1000. if mkey[-3:] == "-fs" else 500.  # Final time (twice the time for fs)
```

Comment:
Why?

Reply:
the truth is that using 500. or 1000. doesn't make difference, the original value can be used.